packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -641,5 +641,27 @@ export default function() {
             this.$element.find('p').first().text('@');
             this.clock.tick(10);
         });
+
+        test('aria-activedescendant should exist on textbox when mention pops up', function(assert) {
```

Comment:
there should be also tests that aria-activedescendant is not set when popup is closed :) 
Seem like this scenario is not working according to the code.